File: Java/linearsearch.java
import java.util.*;

public class linearsearch {

    public static boolean search(ArrayList<Integer> arr, int element) {
        int size = arr.size();
        for (int i = 0; i < size; i++) {
            if (arr.get(i) == element)
                return true;
        }
        return false;
    }

    public static void main(String args[]) {
        Scanner ob = new Scanner(System.in);
        ArrayList<Integer> arr = new ArrayList<Integer>();
        System.out.println("Enter the number of elements you want to enter");
        int size = ob.nextInt();
        int i = 1;
        while (i <= size) {
            System.out.println("Enter the number:");
            arr.add(ob.nextInt());
            i++;
        }
        System.out.println("Enter the element you want to search for");
        int element = ob.nextInt();
        boolean isPresent = search(arr, element);
        if (isPresent)
            System.out.println("The element is present in the list");
        else
            System.out.println("The element is not present in the list");
        return;
    }
}      

File: practice/linear_search.py
def linearSearch(arr: [int], element: int) -> [int]:
    indexes = []
    for index, each in enumerate(arr):
        if each == element:
            indexes.append(index)
    return indexes

print(linearSearch([5, 8, 1, 2, 0, 3, 4, 7, 8, 1, 4, 2, 8, 4, 1, 2], 2))
print(linearSearch([1, 2, 3, 4, 5, 6, 7, 8, 9, 0], 22))
